charts/rookout-hybrid/README.md

```diff
@@ -23,12 +23,37 @@ helm repo update
 
 ### Installation
 
-[This yaml file](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/examples/nginx_lets_encrypt.yaml) shows a basic example configuration using Nginx + cert-manager + Let's Encrypt.
-
-To use it, modify the contents of the example yaml file to match your configuration, and then run:
+To use it, get token from Rookout UI settings > token , and then run:
 
 ```commandline
-helm upgrade --install rookout rookout/rookout-hybrid -f ingerss_example.yaml
+helm upgrade --install rookout rookout/rookout-hybrid \
+    --set rookout.token="YOU_TOKEN_HERE" \
+    --create-namespace rookout
+```
+
+### Datastore access
+
+This installation will not expose your datastore to your web-browser (Rookout client), ingress or other method you are using should expose the datastore service to your web-browser.
```

Comment:
Change to "This installation will not expose your datastore to your web-browser (the Rookout client), unless granted explicitly."